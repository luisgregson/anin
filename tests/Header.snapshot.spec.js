import preact from 'preact';
import Header from '../src/components/header/index';
import render from 'preact-render-to-json';

test('Renders correctly', () => {
  const props = {
    title: 'Test page',
    slugs: ['', 'another-page']
  };
  const header = render(<Header {...props} />);

  expect(header).toMatchSnapshot();
});

