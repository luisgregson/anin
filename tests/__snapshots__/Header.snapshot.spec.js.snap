// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<header
  class="header"
>
  <div
    class="logo"
  >
    <h1
      class="sr-only"
    >
      Test page
    </h1>
    <img
      src="assets/logo.png"
    />
  </div>
  <nav>
    <a
      class="active"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      class=""
      href="/another-page"
      onClick={[Function]}
    >
      Another Page
    </a>
  </nav>
</header>
`;
